worlds/sc2/docs/fr_Starcraft 2.md

```diff
@@ -0,0 +1,87 @@
+# *Starcraft 2*
+
+## Quel est l'effet de la randomisation sur ce jeu ?
+
+Les éléments qui suivent sont les *items* qui sont randomisés et qui doivent être débloqués pour être utilisé dans le 
+jeu:
+1. La capacité de produire des unités, outre les drones/probes/scv.
+2. Des améliorations spécifiques à certaines unités incluant quelques combinaisons qui ne sont pas disponibles dans les 
+campagnes génériques, comme le fait d'avoir les deux types d'évolution en même temps pour une unité *Zerg* et toutes 
+les améliorations de la Lance d'Adun simultanément pour les *Protoss*.
+3. L'accès aux améliorations génériques des unités, e.g. les améliorations d'attaque et d'armure.
+4. D'autres améliorations diverses telles que les améliorations de laboratoire et les mercenaires pour les *Terran*, les 
+niveaux et les améliorations de Kerrigan pour les *Zerg*, et les améliorations de la Lance d'Adun pour les *Protoss*.
+5. Avoir des *minerals*, du *vespene gaz*, et du *supply* au début de chaque mission.
+
+Les *items* sont trouvés en accomplissant du progrès dans les catégories suivantes:
+* Accomplir des missions
```

Comment:
Accomplir -> Completer

Reply:
Completer seems to be discouraged in this context. If the word "Accomplir" seems incorrect, what about "terminer"?

Reply:
How is it discourage?
I think terminer sounds a bit like "you don't have to win, just finish", kind of like the "win" condition on In Utter Darkness.
Still, I'd accept it for missions, but it really doesn't fit for bonus objectives, which then would be better with "Accomplir", although I still prefer "Completer". Either way, not attached to this at all, you can do what you prefer.

Reply:
Using "Compléter" as to finish is an anglicism from "to complete", it seems: https://vitrinelinguistique.oqlf.gouv.qc.ca/21486/les-emprunts-a-langlais/emprunts-semantiques/emploi-deconseille-de-lemprunt-completer

I kept "terminer" for mission and switched to "Réussir" for "objectifs" and "défis", since it felt nice.